BioForm/api/middleware.py

```diff
@@ -0,0 +1,24 @@
+import jwt
+from profiles.models import Account
+from .serializers import LoginSerializer
+from django.utils.deprecation import MiddlewareMixin
+
+
+
+class JTW_AuthenticationMiddleware(MiddlewareMixin):
+    def process_request(self, request):
+        assert hasattr(request, 'COOKIES')
+
+        token = request.COOKIES.get('jwt')
+        
+        try:
+            payload = jwt.decode(token, 'secret', algorithms=['HS256'])
+            user = Account.objects.filter(id=payload['id']).first()
+
+            if user:
+                serializer = LoginSerializer(user)    
```

Comment:
Why do we need to serialize the user?

Reply:
We do not, I have fixed the entire project accordingly.